docs/source/algorithms/CreatePoleFigureTableWorkspace-v1.rst

```diff
@@ -0,0 +1,92 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Create a table with the information required to produce a pole figure, namely, a collection of alphas, betas,
+and intensities.
+
+If a peak parameters table is provided, the intensities and chi squared values are used by the algorithm,
```

Comment:
I think it would be good to describe what columns the input tables are required/expected to have here in the docs (I know you can see from the code example below).